modules/core/src/main/java/org/apache/ignite/cache/query/SqlFieldsQuery.java

```diff
@@ -50,6 +50,16 @@ public class SqlFieldsQuery extends Query<List<?>> {
     /** Default value of the update internal batch size. */
     private static final int DFLT_UPDATE_BATCH_SIZE = 1;
 
+    /**
+     * The minimum priority that a query can have.
+     */
+    public static final int MIN_PRIORITY = 0;
```

Comment:
Let's use full byte range [-128,127]

Reply:
ho ho, i don`t like such a great granularity

Reply:
[-10,10] ?
I just think the one may want to enqueue low-priority task, which will be executed after tasks with default priority.